lib/vfscore/automount.c

```diff
@@ -67,10 +71,47 @@ struct vfscore_volume {
 	unsigned long flags;
 	/* Mount options */
 	const char *opts;
+#if CONFIG_LIBVFSCORE_FSTAB
+	/* Unikraft Mount options, see vfscore_mount_volume() */
+	char *ukopts;
+#endif /* CONFIG_LIBVFSCORE_FSTAB */
 };
 
+#if CONFIG_LIBVFSCORE_FSTAB
+/**
+ * vv->ukopts must follow the pattern below, each option separated by
+ * the character defined through LIBVFSCORE_FSTAB_VOLUME_ARGS_SEP (e.g. with
+ * LIBVFSCORE_FSTAB_VOLUME_ARGS_SEP = ':'):
+ *	[<ukopt1>:<ukopt2>:<ukopt3>:...:<ukoptN>]
+ */
+static int vfscore_volume_process_ukopts(struct vfscore_volume *vv)
+{
+	char *o_curr, *o_next;
+	int rc;
+
+	UK_ASSERT(vv);
+
+	o_curr = vv->ukopts;
+	while (o_curr) {
+		o_next = strchr(o_curr, LIBVFSCORE_FSTAB_VOLUME_ARGS_SEP);
```

Comment:
TODO (for me before merge, so I don't forget): use another separator, different from `LIBVFSCORE_FSTAB_VOLUME_ARGS_SEP` so that they don't conflict.